### ISSUE TITLE
Doesn't work in studio mode when 'duplicate scene' is turned on

### ISSUE THREAD

**Issue by philissimo** [squeek502/obs-txt-trigger#3]
When I work with studio mode and turn on the 'duplicate scene' option, the script is only able to display a source in the preview and not in the live program. (Only after a click on Transition)

Maybe it is possible to trigger the transition or to deactivate 'duplicate scene' display the source and reactivate 'duplicate scene' again. (the same with hiding the source again). I know that this could result in some bad side effects, but it would be great as an option.

**Comment by squeek502** [comment]
I'm not familiar with studio mode, so I might be misunderstanding. If I manually update the visibility of a source it doesn't affect the Program view until I hit transition, so it seems like it not affecting the live version without a transition is intentional?

Are you asking to 'override' this behavior and directly affect the Program view?

**Comment by philissimo** [comment]
Yes, I'm asking for an overwriting of this behavior, because I want to display that in the live program and not only in der Preview.

Thanks a lot.

**Comment by LukasMojzis** [comment]
Also doesn't work when Source is within group. Then it borks the whole OBS (can no longer drag it, ie.)

**Comment by squeek502** [comment]
@lmojzis

> Also doesn't work when Source is within group. Then it borks the whole OBS (can no longer drag it, ie.)

Is this related to studio mode or is this a separate issue?

**Comment by djlee0525** [comment]
Not begin able to drag sources is a separate issue. I'm not sure what causes it, but you can no longer drag a source out of a group or drag a source to change it's position in a list of sources.. Dragging is somehow disabled. If you close the reopen OBS, you can drag sources again.

**Comment by squeek502** [comment]
@djlee0525 @lmojzis moved that to its own issue: #4
